tsconfig.json

```diff
@@ -5,9 +5,10 @@
 		"@tsconfig/strictest/tsconfig.json"
 	],
 	"compilerOptions": {
+		"moduleResolution": "node16",
+		"module": "Node16",
 		"declaration": true,
 		"verbatimModuleSyntax": false,
-		"preserveConstEnums": false,
```

Comment:
This is an important thing to keep for code size and efficiency; the only const enum used in the codebase is internal and kept for readability. Inlining it is better.

Reply:
It can not be enabled while isolated modules is enabled as a note.

Reply:
Isolated modules is not a concern here, it's safe to disable as well if it causes problems.